.storybook/components/DesignTokens/TokensTable.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { InputField, Table, Text } from '../../../src';
+import tokens from '../../data/token-map.json';
+
+export const TokensTable = () => {
+  const [searchTerm, setSearchTerm] = useState('');
+  const tokensList = Object.keys(tokens);
+  // get map of tokens with primary values its mapped tokens
+  const map = {};
+  tokensList.forEach((token) => {
+    const mappedToken = getDeepValue(token);
+    if (!map[mappedToken.token]) {
+      map[mappedToken.token] = {
+        token: mappedToken.token,
+        value: mappedToken.value,
+        mappedTokens: [],
+      };
+    }
+    if (mappedToken.depth) {
+      map[mappedToken.token].mappedTokens.push(token);
+    }
+  });
+  // filter token map based on search term into a list
+  const filteredTokensList = (
+    Object.values(map) as {
+      token: string;
+      value: string;
+      mappedTokens: string[];
+    }[]
+  ).filter(
+    (item) =>
+      // filter based on if token name includes search term or if one of the mapped token names includes search term
+      item.token.includes(searchTerm) ||
+      item.mappedTokens.join('|').includes(searchTerm),
```

Comment:
Some cheating with a delimiter here... think it's easier to read than looping thru the array